### ISSUE TITLE
implement in other network

### ISSUE THREAD

**Issue by sxzy** [AlexHex7/Non-local_pytorch#15]
SO if I want to add nonblock into my own experiment 
I can only add NONLocalBlock2D() into somewhere of the network ? and I did't need to change other params ?

**Comment by AlexHex7** [comment]
Yes, but the size of feature map should not be too small, such as 1x1, 2x2.

**Comment by lc0716** [comment]
> Yes, but the size of feature map should not be too small, such as 1x1, 2x2.

感谢提供的代码！想请教一个问题，我在孪生神经网络两个分支分别加了Non-local模块，其中一个分支的feature map 大小是5*5*512，只要在这个后面加Non-local模块，效果就非常差，另外一个分支是21*21*512，就没有问题，是因为尺寸太小了吗？请问为什么尺寸小效果会不好呢？
